### PR TITLE
[WIP] Added rolebindings to address #7

### PR DESCRIPTION
**Pull request by pranavgaikwad**
Fixes : #7

### CODE REVIEW

**Comment by fbladilo** [comment]
@pranavgaikwad All 3 worked on OCP 3.7 without "new-project" assistance.....

**Comment by jmontleon** [comment]
Doing this:
```
---
apiVersion: v1
kind: Project
metadata:
  labels:
    control-plane: controller-manager
    controller-tools.k8s.io: "1.0"
  name: "openshift-migration"
```

**Comment by jmontleon** [comment]
Doing this:
```
---
apiVersion: v1
kind: Project
metadata:
  labels:
    control-plane: controller-manager
    controller-tools.k8s.io: "1.0"
  name: "openshift-migration"
```
instead of 

```

**Comment by jmontleon** [comment]
Doing this:
```
---
apiVersion: v1
kind: Project
metadata:
  labels:
    control-plane: controller-manager
    controller-tools.k8s.io: "1.0"
  name: "openshift-migration"
```
instead of 

```
---
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
    controller-tools.k8s.io: "1.0"
  name: "openshift-migration"
```
doesn't seem to make a difference. oc new-project seems to be the only way to use the Projects API?

**Comment by jmontleon** [comment]
Well, let me rephrase, there doesn't seem to be a way to use the ProjectRequest API using `oc create -f ...`? You can certainly use curl commands, etc. in addition to oc  new-project...

**Comment by jmontleon** [comment]
https://docs.openshift.com/container-platform/3.7/rest_api/apis-project.openshift.io/v1.ProjectRequest.html
